server/server.go

```diff
@@ -783,7 +783,9 @@ func (s *Server) updateServerINFO(urls []string) bool {
 	}
 
 	// If notification of server additions is always allowed
-	// Set wasUpdated to true so that the notification gets sent.
+	// set wasUpdated to true so that the notification gets sent.
```

Comment:
Remove this line since we are not setting `wasUpdated` here, we are simply returning `true` ;-)
Also, you need to do `go fmt ./...` before pushing your changes otherwise build will fail.

Reply:
Lol, sorry for the runaround.  Still new to Go and this repo.